rpc/src/v1/impls/eth.rs

```diff
@@ -316,7 +316,7 @@ impl<C, S: ?Sized, M, EM> Eth for EthClient<C, S, M, EM> where
 	fn is_mining(&self, params: Params) -> Result<Value, Error> {
 		try!(self.active());
 		match params {
-			Params::None => to_value(&self.external_miner.is_mining()),
+			Params::None => to_value(&(take_weak!(self.miner).is_sealing())),
```

Comment:
it has different semantics - `sealing_work` being enabled means that we are preparing pending blocks, but not that any have actually been used by miners. `is_sealing` actually depends on miners calling the `eth_getWork` RPC or having it delivered to them through HTTP. usually these are more or less the same, but if e.g. we use `--force-sealing` or in the future decide to link pending-preparation with sending own transactions or querying the pending logs, then they're different.